stonesoup/movable/movable.py

```diff
@@ -49,6 +47,45 @@ def __init__(self, *args, **kwargs):
         if not self.states:
             raise ValueError('States must not be empty: it must contain least one state.')
 
+    @staticmethod
+    def from_state(state: State, *args: Any, **kwargs: Any) -> State:
```

Comment:
This method doesn't seem to use any behavior of the Moveable class (and you say its behaviour) is similar to `Prediction` and `Update` classes. 

This method feels like it belongs in a parent class, probably `State`. I'll think about this and see if I can refactor.